web/wp-content/themes/borealis-2021/inc/gutenberg/sidebar-table-of-contents/render-custom-section-block.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ *
+ * Render Page Strip
+ *
+ * @package Trimac
+ * @version 1.0.0
+ * @author  Playground Inc.
+ * @license https://www.gnu.org/licenses/gpl-3.0.txt GNU/GPLv3
+ * @since  1.0.0
+ */
+
+// Check if `register_block_type` exists before calling
+// If Gutenberg isn't enabled it wont exist and error.
+if ( function_exists( 'register_block_type' ) ) {
+    $namespace = pg_get_namespace();
+    register_block_type(
+        $namespace . '/custom-section-block',
+        array(
+            'render_callback' => 'pg_render_custom_section_block',
+        )
+    );
+}
+
+function clean($string) {
```

Comment:
@Braedencraig rather than writing a custom function here I would write this as a utility function in utilities.php (I believe there is already one called pg_slugify() that you can use for now. You should be able to call the pg_slugify() function from within your render.